elastic/apm-server
# Add metric_type and unit to system metrics

## Motivation/summary

Add `metric_type` and `unit` to system metric fields, so Kibana will be able to aggregate and format them appropriately.

Also update `apmpackage/cmd/gen-package` to copy across these field properties.

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. `apm-server export template`
2. Check the template includes `metric_type` and `unit` field metadata for `system.*` metrics

## Related issues

https://github.com/elastic/apm-server/issues/4487
